.github/workflows/openmldb-docker.yml

```diff
@@ -68,4 +70,5 @@ jobs:
           platforms: linux/amd64
           tags: |
             ${{ env.DOCKERHUB_TAG }}
-          build-args: OPENMLDB_VERSION=${{ github.event.inputs.OPENMLDB_VERSION }}
\ No newline at end of file
+            ${{ env.DOCKERHUB_LATEST }}
```

Comment:
strange, why not just pass `OPENMLDB_VERSIOn` = `latest`

Reply:
if pass `OPENMLDB_VERSION`, we should trigger docker release twice. And we may forget to release the latest version

Reply:
what is the meaning of `latest` ?  If that always link to a const number tag, it may not necessary.

Reply:
and if we are gona patch a previous version, the latest tag also overrides